src/config/serverconfigreader.cpp

```diff
@@ -508,6 +510,16 @@ po::options_description ServerConfigReader::_defineConfigOptions()
         po::value<std::string>( &(_vars["ExperimentalTapeRESTAPI"]) )->default_value("false"),
         "Enable or disable experimental features of the TAPE REST API"
     )
+    (
+        "TransferServiceAllocatorAlgorithm",
+        po::value<std::string>( &(_vars["TransferServiceAllocatorAlgorithm"]) )->default_value(FTS3_CONFIG_SERVERCONFIG_TRANSFER_SERVICE_ALLOCATOR_ALGORITHM),
+        "Specify transfer service's slot allocation algorithm as MAXIMUM_FLOW or GREEDY"
```

Comment:
nit: 
... MAXIMUM_FLOW or GREEDY (default)"
... DEFICIT or RANDOMIZED (default)"
(consistent order and mention of default option)